public/app/features/templating/template_srv.ts

```diff
@@ -347,6 +349,55 @@ export class TemplateSrv implements BaseTemplateSrv {
     });
   }
 
+  getVariablesMapInTemplate(
```

Comment:
What do you think about the following to keep a consistent naming convention with the other functions?

```suggestion
  getVariablesInTarget(
```